freeciv-web/src/main/webapp/javascript/client_main.js

```diff
@@ -38,45 +38,47 @@ function set_client_state(newstate)
     civclient_state = newstate;
 
     switch (civclient_state) {
-    case C_S_RUNNING:
-      clear_chatbox();
-      $.unblockUI();
-      show_new_game_message();
+      case C_S_RUNNING:
```

Comment:
Where possible follow https://github.com/freeciv/freeciv/blob/master/doc/CodingStyle#L76. No need for this indent.